client/driver/docker.go

```diff
@@ -38,10 +38,7 @@ import (
 )
 
 var (
-	// We store the clients globally to cache the connection to the docker daemon.
-	createClients sync.Once
```

Comment:
Since we're removing this we'll need to replace it with a lock so multiple goroutines don't try to read/write the global client vars at once. sync.Once did this locking for us.